Sources/Apollo/HTTPNetworkTransport.swift

```diff
@@ -270,4 +281,26 @@ public class HTTPNetworkTransport: NetworkTransport {
     
     return ["query": operation.queryDocument, "variables": operation.variables]
   }
+
+  private func requestMultipartFormData<Operation: GraphQLOperation>(for operation: Operation, files: [GraphQLFile]) -> MultipartFormData {
+	    let formData = MultipartFormData()
+
+	    let fields = requestBody(for: operation)
+	    for (name, data) in fields {
+	      if let data = data as? GraphQLMap {
+	        let data = try! serializationFormat.serialize(value: data)
+	        formData.appendPart(data: data, name: name)
+	      } else if let data = data as? String {
+	        formData.appendPart(string: data, name: name)
+	      } else {
+	        formData.appendPart(string: data.debugDescription, name: name)
+	      }
+	    }
+
+	    for f in files {
```

Comment:
a) This would probably be more readable as a `files.forEach` loop
b) Please don't just use single letters on things like this - if you want to use a `$0` in the `forEach` closure that's fine, but if you're gonna name something, please give it a name that is a full word. 😄